test/functional/ex_cmds/oldfiles_spec.lua

```diff
@@ -1,49 +1,92 @@
-local h = require('test.functional.helpers')
+local Helpers = require('test.functional.helpers')
+local Screen  = require('test.functional.ui.screen')
 
-local buf     = h.curbufmeths
-local command = h.command
-local eq      = h.eq
-local execute = h.execute
-local feed    = h.feed
-local nvim    = h.nvim
+local buf         = Helpers.curbufmeths
+local command     = Helpers.command
+local eq          = Helpers.eq
+local execute     = Helpers.execute
+local feed        = Helpers.feed
+local nvim        = Helpers.nvim
+local nvim_prog   = Helpers.nvim_prog
+local set_session = Helpers.set_session
+local spawn       = Helpers.spawn
```

Comment:
I agree it's not good, we're going to clean up our Lua code soon. We need to put a linter in place and write style guide, first (we can use your guide as a base).

> Maybe we can use this PR as a showpiece

Maybe, but it could hold up this PR for awhile. I have not found any authority on Lua best practices.